docs/source/mistral_yaql.rst

```diff
@@ -1,326 +0,0 @@
-engine. The context contains all the workflow inputs, published variables, and result of completed
-tasks up to this point of workflow execution, including the current task. The YAQL expression can
-refer to one or more variables in the context. The reserved symbol ``$`` is used to reference the
-context. For example, given the context ``{"ip": "127.0.0.1", "port": 8080}``, the string
-``https://<% $.ip %>:<% $.port %>/api`` returns ``https://127.0.0.1:8080/api``. The following is the
-same example used in a workflow:
-
-.. code-block:: yaml
-
-    version: '2.0'
-
-    examples.yaql-basic:
-        type: direct
-        input:
-            - ip
-            - port
-        tasks:
-            task1:
-                action: examples.call-api
-                input:
-                    endpoint: https://<% $.ip %>:<% $.port %>/api
-
-Certain statements in Mistral such as on-success and on-error can evaluate boolean logic. The
-``on-condition`` related statements are used for transition from one task to another. If a
-boolean logic is defined with these statements, it can be used to evaluate whether the transition
-should continue or not. Complex boolean logic using a combination of ``not``, ``and``, ``or``, and
-parentheses is possible. Take the following workflow as an example: Execution of certain branch
-in the workflow depends on the value of ``$.path``. If ``$.path = a``, then task ``a`` is executed.
-If ``$.path = b``, then task ``b``. Finally task ``c`` is executed if neither.
-
-.. literalinclude:: /../../st2/contrib/examples/actions/workflows/mistral-branching.yaml
-   :language: yaml
-
-The statement ``with-items`` in Mistral is used to execute an action over iteration of one or more
-list of items. The following is a sample Mistral workflow that iterates over the list of given names
-to invoke the action to create individual VM.
-
-.. code-block:: yaml
-
-    version: '2.0'
-
-    examples.create-vms:
-        type: direct
-        input:
-            - names
-        tasks:
-            task1:
-                with-items: name in <% $.names %>
-                action: examples.create-vm
-                input:
-                    name: <% $.name %>
-
-``with-items`` can take more than one list as the following example illustrates. In this case,
-a list of VMs and IP addresses are passed as inputs and then iterated through step by step together.
-
-.. code-block:: yaml
-
-    version: '2.0'
-
-    examples.create-vms:
-        type: direct
-        input:
-            - names
-            - ips
-        tasks:
-            task1:
-                with-items:
-                    - name in <% $.names %>
-                    - ip in <% $.ips %>
-                action: examples.create-vm
-                input:
-                    name: <% $.name %>
-                    ip: <% $.ip %>
-
-The sections below contain additional YAQL examples of how to work with lists and dictionaries,
```

Comment:
There is a lot of useful general info here on YAQL. In the Orquesta pages the approach seems to have been to just point to the YAQL and Jinja projects themselves?

Are we happy to remove the pages and just refer to the YAQL/Jinja projects or want to put a page together for Orquesta/YAQL and Orquesta/Jinja?

Reply:
I would vote for a page for Orquesta/YAQL and Orquesta/Jinja. But this shouldn't be a blocker for this PR imho.